### DISCUSSION TITLE
Reboot NSPanel from the display?

### DISCUSSION THREAD

**Discussion by rasmus-wi** [Blackymas/NSPanel_HA_Blueprint#1708] [Q&A] [answered]
Is there a way to reboot a NSPanel that has lost its wifi connection? It happened to me Yesterday, and I realized that without wifi, the button pages, the entities pages, and most other pages were unavailable. I see some mention of a display settings page, which is supposed to have a reset button, but no mention of how to access that page, and I haven't been able to figure out how to reach it. Fortunately It did reconnect when I restarted my router, and it's not yet installed in the wall, so it's at the moment pretty easy for me to power cycle it. Once installed in the wall I will have to power cycle one of the fuses in my apartment if I need to power cycle an NSPanel.

**Comment by Doughboy68** [comment, marked as answer]
Can long press the clock and it has screen allowing to restart.
Also if panel is not accessible via screen, can long press the right hardware button for 10 seconds or longer until it reboots.
